.github/workflows/create-upmerge-PRs.yml

```diff
@@ -0,0 +1,67 @@
+name: Zephyr upmerge
+
+#on:
+#  schedule:
+#    - cron: "0 0 * * *"
+on: workflow_dispatch
+
+env:
+  GH_TOKEN: ${{ secrets.NCS_GITHUB_UPMERGE_TOKEN }}
+  ZEPHYR_UPSTREAM: https://github.com/zephyrproject-rtos/zephyr
+
+jobs:
+  auto-upmerge-create-PRs:
+    if: github.repository == 'nrfconnect/sdk-nrf'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v3
+        with:
+          path: ncs/nrf
+          fetch-depth: 0
+
+      - name: Install base dependencies
+        working-directory: ncs
+        run: |
+          sudo pip3 install -U setuptools wheel pip
+          pip3 install -r nrf/scripts/requirements-base.txt
+          pip3 install -r nrf/scripts/requirements-extra.txt
+
+      - name: West init and update
+        working-directory: ncs
+        run: |
+          west init -l nrf
+          west update -n -o=--depth=1 zephyr
+          git config --global user.email "noreply@nordicsemi.no"
+          git config --global user.name "NordicBuilder"
+          echo SDK_ZEPHYR=$(west list zephyr -f {url} | awk -F// '{print $NF}') >> $GITHUB_ENV
```

Comment:
Low risk but `$GITHUB_ENV` as a path should be enclosed in "", just in case space any directory name in path contains space.